src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -6,19 +6,16 @@
 import java.util.Locale;
 
 import redis.clients.jedis.commands.ProtocolCommand;
-import redis.clients.jedis.exceptions.JedisAskDataException;
-import redis.clients.jedis.exceptions.JedisBusyException;
-import redis.clients.jedis.exceptions.JedisClusterException;
-import redis.clients.jedis.exceptions.JedisConnectionException;
-import redis.clients.jedis.exceptions.JedisDataException;
-import redis.clients.jedis.exceptions.JedisMovedDataException;
-import redis.clients.jedis.exceptions.JedisNoScriptException;
+import redis.clients.jedis.exceptions.*;
 import redis.clients.jedis.util.RedisInputStream;
 import redis.clients.jedis.util.RedisOutputStream;
 import redis.clients.jedis.util.SafeEncoder;
 
 public final class Protocol {
 
+  public static final String WRONGPASS_PREFIX = "WRONGPASS";
+  public static final String NOPERM_PREFIX = "NOPERM";
+
```

Comment:
Please add these prefixes right after existing prefixes.

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0